hypervisor/arch/x86/cpu_caps.c

```diff
@@ -118,7 +107,7 @@ bool is_apl_platform(void)
 {
 	bool ret = false;
 
-	if ((boot_cpu_data.displayfamily == 0x6U) && (boot_cpu_data.displaymodel == 0x92U)) {
+	if ((boot_cpu_data.displayfamily == 0x6U) && (boot_cpu_data.displaymodel == 0x5c)) {
```

Comment:
Any reason for this change? It is completely different.
BTW 0x5c -->0x5CU ?

Reply:
Hi, Eddie.
From SDM,
`06_5CH Intel Atom processors based on Goldmont Microarchitecture`
APL platform's DisplayModel is 0x5c, `0x92U` was a bug.

I will change 0x5c -->0x5cU